thrift_base.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import os
+from conans import ConanFile, tools
+
+
+class ThriftBase(ConanFile):
+    version = "0.12.0"
+    description = "Thrift is an associated code generation mechanism for RPC"
+    url = "https://github.com/helmesjo/conan-thrift"
```

Comment:
This isn't your mistake. I have forgotten to adapt that
```suggestion
    url = "https://github.com/bincrafters/conan-thrift"
```

Reply:
done!